bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogger.java

```diff
@@ -1000,6 +1000,11 @@ public void scanEntryLog(long entryLogId, EntryLogScanner scanner) throws IOExce
                 long offset = pos;
                 pos += 4;
                 int entrySize = headerBuffer.readInt();
+                if (entrySize <= 0) { // hitting padding
```

Comment:
Is it possible the entrySize is normal but the ledgerId is abnormal?

Reply:
Skip and continue to read the next entry.
See "scanner.accept(ledgerId)" in line 1011.
The code is consistent with version 4.16.

Reply:
I think it can't resolve the disorder problem. After this pr, could you parse the entry file incorrectly?